### PR TITLE
Add and remove

### PR DESCRIPTION
**Pull request by Kaghenimbale**
- [ ]  Implement a function for adding a new task
- [ ]  Implement a function for deleting a task
- [ ]  Implement a function for editing task descriptions.

### CODE REVIEW

File: src/module/add.js [hunk 1]
```diff
@@ -1,22 +1,21 @@
-import renderToDo from './display.js';
+// import { UI } from './storeData.js'
+// export const renderDomContentDb = () => {
+//     const newData = JSON.parse(localStorage.getItem('data') || '[]');
```
Comment (mukhammadrizooff) [inline src/module/add.js]: - [ ] [REQUIRED] Please delete `unnecessary` comments from your codebase so your codebase will be cleaner 🙏🏼

File: src/module/add.js [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
 
-const form = document.getElementById('form');
+//     form.addEventListener('submit', (e) => {
+//       e.preventDefault();
+//       const InputValue = document.querySelector('#input-list').value;
```
Comment (mukhammadrizooff) [inline src/module/add.js]: - [ ] [REQUIRED] Please delete `unnecessary` comments from your codebase so your codebase will be cleaner 👍🏼

File: src/module/add.js [hunk 3]
```diff
@@ -1,22 +1,21 @@
...
 
-const addToDo = (e) => {
-  const newData = JSON.parse(localStorage.getItem('data') || '[]');
-  e.preventDefault();
-  const InputValue = document.querySelector('#input-list').value;
+//       const dataObj = {
+//         description: InputValue,
+//         completed: false,
+//         index: newData.length,
+//       };
```
Comment (mukhammadrizooff) [inline src/module/add.js]: - [ ] [REQUIRED] Please delete `unnecessary` comments from your codebase so your codebase will be cleaner 🙏🏼

File: src/module/add.js [hunk 4]
```diff
@@ -1,22 +1,21 @@
...
 
-  const dataObj = {
-    description: InputValue,
-    completed: false,
-    index: newData.length,
-  };
+//       const userINterface = new UI(newData);
+//       userINterface.pushData(dataObj);
+//       userINterface.readData();
```
Comment (mukhammadrizooff) [inline src/module/add.js]: - [ ] [REQUIRED] Please delete `unnecessary` comments from your codebase so your codebase will be cleaner 👍🏼

File: src/module/add.js [hunk 5]
```diff
@@ -1,22 +1,21 @@
...
-  newData.push(dataObj);
-  localStorage.setItem('data', JSON.stringify(newData));
-  renderToDo();
-  form.reset();
-};
-
-export default addToDo;
+//       form.reset();
+//     });
+// }
```
Comment (mukhammadrizooff) [inline src/module/add.js]: - [ ] [REQUIRED] Please delete `unnecessary` comments from your codebase so your codebase will be cleaner 🙏🏼

File: src/module/storeData.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+//     this.newData.push(dataObj);
+//     localStorage.setItem('data', JSON.stringify(this.newData));
+//   }
+
+//   readData() {
+//     const newData = JSON.parse(localStorage.getItem('data') || '[]');
+//     console.log(newData);
+//     this.pushData()
+//   }
+// }
```
Comment (mukhammadrizooff) [inline src/module/storeData.js]: - [ ] [REQUIRED] Please delete `unnecessary` comments from your codebase so your codebase will be cleaner 👍🏼

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Kaghenimbale, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 
- Correct `GitHub Flow` ✔️ 
- Project working well ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
### Hi @Kaghenimbale, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- Function for adding a new book is implemented. ✔️
- Function for remove a book from the array is implemented. ✔️
- Function for editing book descriptions is implemented corectly. ✔️
- All changes to the To-Do List  are saved in local storage. :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] When I click the link to the `MIT.md` file that you added, I get this 404 `error`.  Please update your Readme, it will be nice to add and update this section(`License`) with the type of License and a link to the license file, as this will make your readme more informative and professional.  👍🏻 

- [OPTIONAL] It's one of the project requiements to create a new JavaScript file for the new functionality. Kindly address this ✍🏻 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**